activejob/test/jobs/continuable_duplicate_step_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ContinuableDuplicateStepJob < ActiveJob::Base
+  include ActiveJob::Continuable
+
+  def perform
+    step :duplicate do |step|
+    end
+    step :duplicate do |step|
+    end
+  end
+end
```

Comment:
All these "fixture" jobs could be defined inside the test classes themselves.

Reply:
Thanks. IMHO it's much easier to understand tests when the job they rely on is just above.